crates/bevy_sprite/src/render/sprite_sheet.vert

```diff
@@ -38,7 +38,7 @@ layout(set = 2, binding = 1) uniform TextureAtlasSprite {
 void main() {
     Rect sprite_rect = Textures[TextureAtlasSprite_index];
     vec2 sprite_dimensions = sprite_rect.end - sprite_rect.begin;
-    vec3 vertex_position = vec3(Vertex_Position.xy * sprite_dimensions, 0.0);
+    vec3 Vertex_Position = vec3(Vertex_Position.xy * sprite_dimensions, 0.0);
```

Comment:
I'm not convinced this is an improvement. The new "vertex position" is semantically different than the input Vertex_Position. I think overloading the vertex attribute makes it harder to understand.

Reply:
Oh thats a leftover! (Was trying out a different naming convention for some time)